### PR TITLE
[WIP] Add webpacker

### PR DESCRIPTION
**Pull request by nerdofthunder**
Need to see how to make sure this works with the webserver, which I believe is Unicorn.

### CODE REVIEW

**Comment by sethherr** [comment]
I’ll check this out tomorrow!

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* eslint no-console:0 */
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+//
+// To reference this file, add <%= javascript_pack_tag 'application' %> to the appropriate
+// layout file, like app/views/layouts/application.html.erb
+
+console.log('Hello World from Webpacker')
```
Comment (nerdofthunder) [inline app/javascript/packs/application.js]: This file probably doesn't need to be there.

**Comment by sethherr** [comment]
I'm going to make updates in a separate PR to merge into this.

Sorry for the delay in checking this out, thanks for getting it started!

File: app/javascript/packs/application.js (hunk 1 above)
Comment (sethherr) [reply app/javascript/packs/application.js]: True - but we might as well have something to make sure we're checking that the pack loads correctly. 

Leaving something similar in, modifying the specific message though 😉

**Comment by sethherr** [comment]
Closing. I added a few things in #464 and am merging that.

**Comment by sethherr** [comment]
It worked! Webpacker is deployed into production!

Thanks @nerdofthunder 🙂

**Pull request closed without merging**